lib/src/utils.dart

```diff
@@ -32,6 +32,9 @@ Patch patchAfter(AstNode node, String text) {
   return Patch(end.file.span(end.offset, end.offset), text);
 }
 
+/// Returns true if [map] does not contain any duplicate values.
+bool valuesAreUnique(Map map) => map.values.toSet().length == map.length;
```

Comment:
```suggestion
bool valuesAreUnique(Map<Object, Object> map) => map.values.toSet().length == map.length;
```

It's good practice to explicitly write type arguments, even when they're just `Object`. If/when we turn on the Dart analyzer's `no-implicit-dynamic` flag, this will become mandatory.

Reply:
Done